Scripts/Runtime/Util/Systems/WorldCacheUtil.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Anvil.Unity.DOTS.Util
+{
+    /// <summary>
+    /// Helper class for managing a cached view of a <see cref="World"/>
+    /// </summary>
+    public static class WorldCacheUtil
```

Comment:
Thoughts on naming this something like `WorldCacheLookup`? Static Util types are typically stateless and this is rally just a holder for the lookup.

Reply:
Renamed to `WorldCacheManager` to match the other name changes on the Util classes.

Reply:
Not relevant anymore due to Systems refactor